cluster/gce/config-default.sh

```diff
@@ -259,6 +259,7 @@ fi
 # Optional: Install cluster DNS.
 # Set CLUSTER_DNS_CORE_DNS to 'false' to install kube-dns instead of CoreDNS.
 CLUSTER_DNS_CORE_DNS="${CLUSTER_DNS_CORE_DNS:-true}"
+CORE_DNS_VERSION="${KUBE_CORE_DNS_VERSION:-}"
```

Comment:
if KUBE_CORE_DNS_VERSION is unset, will this produce an invalid manifest? If so, we should either set a valid default here or remove the `:-` to make this produce an error if `KUBE_CORE_DNS_VERSION` is not set

Reply:
I don't see `KUBE_CORE_DNS_VERSION` specified in the repo. Won't that make kube-up for GCE break?

Reply:
It's not required -- the default value is used in configure-helper.sh if this is not specified.

In https://github.com/kubernetes/kubernetes/pull/77918 I was adding similar config variable using slightly different approach:
* add variable here with default value
* in configure-helper.sh don't use default, but apparently this made gke test failing (see https://github.com/kubernetes/kubernetes/pull/78010/files)

This time I'm using slightly different approach to avoid breaking gke.